index.html

```diff
@@ -0,0 +1,99 @@
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="stylesheet" type="text/css" href="css/index.css">
+  <script src="https://kit.fontawesome.com/37732951b9.js" crossorigin="anonymous"></script>
+  <title>Hacked News</title>
+  <link rel="shortcut icon" href="https://img.icons8.com/windows/50/000000/hacker-news.png">
+</head>
+
+<body>
+  <header class="homePageHeader">
+    <nav>
+      <div class="appTitle">
+        <h1>Hacked News</h1>
+      </div>
+      <div class="navLinks">
+        <a href="about.html">About</a>
+        <a href="https://reacthacker-news.herokuapp.com/LogIn" class="navLogInButton">Log In</a>
+      </div>
+    </nav>
+    <div class="headerTextAndButton">
+      <h1>Bringing Hacker News to the Modern Era</h1>
+      <a href="https://reacthacker-news.herokuapp.com/SignUp" class="headerSignUpButton">Sign Up</a>
+    </div>
+  </header>
+  <div class="contentWidth">
+    <main class="homePageMain">
+      <section>
+        <h2>Hacker News, But Better</h2>
+        <div class="sectionContent">
+          <p>Y Combinators "Hacker News" is a great source of news and information, but its design is. . . well, dated.
+            Hacked News is designed to fix that problem, with a responsive and modern look that will be a joy to use on
+            all your devices, no matter the screen size.</p>
+          <img src="img/responsive.jpeg" alt="person typing on laptop">
+        </div>
+      </section>
+      <section>
+        <h2>News for Programmers</h2>
+        <div class="sectionContent sectionMiddle">
+          <p>At our core, Hacked News is still providing you access to all the latest and greatest news about
+            programming and technology. With the Hacked News web app, you don't have to give up on either a modern
+            design, or the most current and important news that affects you.</p>
+          <img src="img/code.jpeg" alt="lines of code">
+        </div>
+      </section>
+      <section>
+        <h2>Save Time</h2>
+        <div class="sectionContent">
+          <p>But it's not all about aesthetics, Hacked News's design is also more functional and efficent. Hacked News's
+            improved readablity will save you time finding the information you need, and our default view showing you
+            just the top 20 stories ensures you aren't bogged down with less important headlines.</p>
+          <img src="img/time.jpeg" alt="person holding watch in hand">
+        </div>
+      </section>
+    </main>
+  </div>
+  <footer>
+    <div class="footerMenu">
+      <div class="footerMenuRow">
+        <div class="footerConnect">
+          <h3>Connect</h3>
+          <div class="connectIconsRow">
+            <a href="https://github.com/Build-Week-React-HackerNews-Clone" target="_blank"><i
+                class="fab fa-github-square"></i></a>
+            <a href="https://twitter.com" target="_blank"><i class="fab fa-twitter-square"></i></a>
```

Comment:
I would consider removing the Facebook, Twitter, and Medium links as they only link to the home pages for each site. Maybe direct users to contact developers through Twitter or GitHub as found on the About page?